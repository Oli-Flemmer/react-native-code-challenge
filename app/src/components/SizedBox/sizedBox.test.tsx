import React from 'react';
import SizedBox, { Props } from './index';
import { render as testRender, screen } from '@testing-library/react-native';


it('renders correctly of SizedBox', () => {
	const render = (props?: Partial<Props>) =>
		testRender(<SizedBox width={40} height={40} {...props} />);
	render();
	expect(screen.toJSON).toMatchSnapshot();

});
