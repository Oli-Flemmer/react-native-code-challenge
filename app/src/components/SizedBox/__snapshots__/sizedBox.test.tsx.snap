// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly of SizedBox 1`] = `[Function]`;
